components/SStatus.vue

```diff
@@ -1,17 +1,51 @@
 <template>
   <div class="s-status">
-    <v-chip
-      v-for="(item, key) in _items"
+    <v-card
+      v-for="(verification, key) in _verified"
       :key="key"
-      class="mr-2 mb-2"
+      class="mr-2 mb-2 verified"
     >
-      {{ item.name }}
-    </v-chip>
+      <div>
+        <img class="badge-image" v-bind:src="'data:image/png;base64,' + verification.base64"/>
```

Comment:
Since the openbadges standard specifies that it'll be PNG images and this is more than just the MIME type, we probably won't be pulling in a dependency for this.